athena-dynamodb/pom.xml

```diff
@@ -80,6 +80,19 @@
             <version>${log4j2Version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>athena-federation-integ-test</artifactId>
+            <version>1.0</version>
+            <scope>test</scope>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/software.amazon.awscdk/dynamodb -->
+        <dependency>
+            <groupId>software.amazon.awscdk</groupId>
+            <artifactId>dynamodb</artifactId>
+            <version>${aws-cdk.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this needed? isn't it inherited from the athena-federation-integ-test module?

Reply:
This one is not inherited. The `dynamodb` artifact is specific to the DynamoDB connector so it only exists in this pom.

Reply:
ahhh, this is the DDB module of CDk. got it.